NOTICE.txt

```diff
@@ -0,0 +1,43 @@
+
+                            The AsyncHTTPClient Project
+                            ===========================
+
+Please visit the AsyncHTTPClient web site for more information:
+
+  * https://github.com/swift-server/async-http-client
+
+Copyright 2017-2021 The AsyncHTTPClient Project
```

Comment:
Does AHC really date back to 2017?! 🤯

Reply:
The first commit is from 2019: https://github.com/swift-server/async-http-client/commit/da770caf2e7878081b6444f03a606728a78156ac

However, they already include files with license headers which state 2017-2018 e.g: https://github.com/swift-server/async-http-client/blob/da770caf2e7878081b6444f03a606728a78156ac/Sources/SwiftNIOHTTP/Utils.swift#L5

So I don't really know which year I should choose.

Reply:
2017.